cli/src/cmd/root.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/luno/jettison/log"
+	"github.com/spf13/cobra"
+
+	"cli/cmd/commands"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "cli",
+	Short: "A cli to assist with package deployment and management",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		log.Error(context.Background(), err)
+		panic(err)
+	}
+}
+
+func init() {
+	// TODO: read the docs for cobra.OnInitialize() and decide if it's needed
+	cobra.OnInitialize()
```

Comment:
OnInitalize() accepts a function that loads a config (like env vars), here's an example :
https://github.com/schadokar/my-calc/blob/master/cmd/root.go

So either we add such function, or we can simply remove it, wdyt ?

Reply:
We don't need a function like this.  The config files and environments are handled in functions as part of the preparing launch function. Removed it in [e27cde6](https://github.com/openhie/package-starter-kit/pull/31/commits/e27cde68cd508bdc45a7f80344f86c634dd7a981)